.gitignore

```diff
@@ -128,6 +128,10 @@ coverage/
 # Symbols
 app.*.symbols
 
+# Platform specific in the examples
+examples/api/android/
```

Comment:
Lets put these exclusions in the `/examples/api` directory as a new .gitignore file. The exclusions in this file are ones that should apply to anywhere in the codebase. 

When moved the contents would be 
```
# Unused platform specific files
android/
ios/
```